roles/pulibrary.nginxplus/files/conf/http/templates/libwww-proxy-pass-prod.conf

```diff
@@ -0,0 +1,6 @@
+    rewrite ^/locator/(.*)$ https://locator-prod.princeton.edu/$1 redirect;
+    
+    rewrite ^/special_collections/mudd-dbs/(.*)$ /mudd-dbs/$1 redirect;
+    location /mudd-dbs {
+        proxy_pass http://libruby-prod.princeton.edu/mudd-dbs/;
```

Comment:
will this be modified with a future PR?

Reply:
I expected this to be `https://lib-mudd-prod.princeton.edu/`

Reply:
@kayiwa yes, once we get the green light that staging is working as expected we will modify the production path to the new prod machine.